examples/withContainerDimensions.js

```diff
@@ -23,18 +23,15 @@ const styles = StyleSheet.create({
 // Our original component is provided with original props and it will
 // additionally receive width and height once rendered inside grid.
 // Default values are fallback if rendered outside grid.
-const Info = ({ width, height }) => (
-  <Text style={styles.text}>{width}pt x {height}pt</Text>
+const Info = () => (
+  <WithContainerDimensions>{(width, height) => (<Text>{width}pt x {height}pt</Text>)}</WithContainerDimensions>
```

Comment:
The same as WithSizeClass;
It might make more sense to extract this into component `InfoFaCC` and change the other one to the `InfoHOC`, that way it would be clear those components are the same just different syntax.